TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utilities/PullUp.java

```diff
@@ -0,0 +1,91 @@
+package org.firstinspires.ftc.teamcode.utilities;
+
+import static java.lang.Double.max;
+import static java.lang.Double.min;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+public class PullUp {
+    public DcMotor pullUpMotor1;
+    public DcMotor pullUpMotor2;
+    public static double Encoder_Ticks = 537.7;
```

Comment:
minor grievance but please make this ENCODER_TICKS because thats how constants should be smh